src/schemes/fluid/weakly_compressible_sph/density_diffusion.jl

```diff
@@ -23,7 +23,7 @@ end
 @doc raw"""
     DensityDiffusionMolteniColagrossi(; delta)
 
-The commonly used density diffusion term by Molteni & Colagrossi (2009).
+The commonly used density diffusion term by [Molteni2009](@cite).
```

Comment:
When part of a sentence, please cite as text.
Same for the other density diffusion formulations.